src/main/java/com/ibm/crypto/plus/provider/ECUtils.java

```diff
@@ -254,4 +257,44 @@ static String bytesToHex(byte[] bytes) {
         return new String(hexChars);
     }
 
+     /**
```

Comment:
Indentation here is slightly different then the Java 17 PR. Please make this match.

Reply:
updated